markdown/chapters/entombsday/playing-the-deck.md

```diff
@@ -0,0 +1,252 @@
+
+It can also be correct to _transform_ back and forth or stay Tin Fins in game two and only _transform_ for game 3.
+
+For example, when playing against Delver, you should swap to Doomsday in game two to try and sneak a win by surprising them. If there is a game three, it's defensible to use either plan, depending on what graveyard hate you see in game two. The Tin Fins plan is stronger again Delver in general and can beat a Surgical Extraction, but if you see permanent hate, such as :card[Grafdigger's Cage], it may be better to stay on the Doomsday plan.
+
+When playing in paper, you should shuffle all 15 sideboard cards in between each game, so the opponent doesn't know which gameplan you are on.
+
+### Partial Sideboarding
+
+In some matchups&mdash;such as Dredge, Hogaak, and Turbo Depths&mdash;where you expect a fast clock, discard, but no countermagic you can use a hybrid sideboard plan to create a "two-card monte"-style deck. The idea here is to have a higher density of "action" spells, allowing for a higher number of turn 1 combos.
+
+The sideboard plan for this is generally:
+
+| **In**                    | **Out**                 |
+| ------------------------- | ----------------------- |
+| 4x :card[Doomsday]        | 4x :card[Force of Will] |
+| 1x :card[Edge of Autumn]  | 2x :card[Daze]          |
+| 1x :card[Shelldock Isle]  | 2x :card[Preordain]     |
+| 2x :card[Street Wraith]   | 1x :card[Griselbrand]   |
+| 1x :card[Thassa's Oracle] |                         |
+
+## Common Doomsday Piles
+
+Because the sideboard swap is so clean, Entombsday has very few deck-specific piles that aren't also available to Turbo Doomsday lists. I would again recommend reading through the resources available in the [Turbo Doomsday section][meandeck:basics] of the Wiki. As a quick reference, some common piles to know are:
+
+> Pass the Turn OR Cantrip + Land Drop
+
+:::row{variant=PILE}
+
+- SI
+- Em
+- TO
+- TO
+- CoS
+  :::
+
+This is a variation on the Shelldock + Emrakul pile that provides three win attempts over multiple turns and plays around things such as :card[Endurance] by putting very few cards into your graveyard. A few things to note with this pile are:
+
+1. It can be sped up by a turn if you have access to a cantrip and a land drop.
+2. You don't always need to choose Emrakul for the Hideaway trigger. When playing against Painter, it is sometimes correct to leave Emrakul in your deck and hide Thassa's Oracle under the Shelldock Isle, allowing you to cast Oracle before your Draw Step after being milled out.
+
+> Cantrip + {U} + X
+
+:::row{variant=PILE}
```

Comment:
Piles not showing correctly, otherwise all good.

Reply:
Fixed syntax